### ISSUE TITLE
Add contribution option

### ISSUE THREAD

**Issue by tyler-hitzeman** [focus-launcher/focus-launcher-android#16]
Setup a way for a user to contribute to the project from within the app.

Siempo's contribution page was closed, so when a user tries to contribute, they see an error saying "The item you requested is not available for purchase."

**Comment by tyler-hitzeman** [comment]
Open Collective seems like the best option for this, as it lets our outsource a ton of admin stuff. 
We need 100 stars before we can setup an account, so this is on hold until then.
